### PR TITLE
Update - persistent and non-persistent terminology clarity

### PR DESCRIPTION
**Pull request by dacarroll**
Update azure-virtual-desktop-content.md

added note to include clarity around how AVD functions in reference to the key terminology used within VDI infrastructure.

Before selecting the "Create pull request" button:  

1. Enter a meaningful title above^, using a prefix if necessary and keywords "New" or "Update" indicating the nature of changes.

2. Describe the summary, scope, and intent of this PR:  
To create clarity about terminology usage of persistent and non-persistent VDI as it pertains to Azure Virtual Desktop. 

3. Insert links(s) to any related work item(s) or supporting detail:  
  

  

  
<details><summary>AFTER YOUR PR HAS BEEN CREATED, expand this section for tips and additional instructions.</summary>    
      

These are common guidelines for contributions across the repos managed by the Cloud Architecture Content Team (CACT). Some repositories may have additional specific requirements that are not listed here.   

## Guidance for all contributors  
  
  | **Topic** | **Guidance** |
  | ----------| ------------ |
  | **Draft PR** | If your PR will be a work-in-progress for more than a day or two, select the **Convert to draft** link in the upper right of the page (under **Reviewers**) to change it to a draft. For future reference, you can also do this using the **Create pull request** button drop-down during PR creation. | 
  | **ms.date metadata** | <ul><li>Don't update an article's "ms.date" metadata property unless you've done a **full freshness review** of the content. A full freshness review includes changes required to correct or improve the **full** technical accuracy of the article.</li><li>Don't update "ms.date" if you're doing targeted changes to improve non-technical aspects of the article, such as editorial quality, art improvements, article template alignment, etc.</li><li>If you've changed any "ms.date" properties for work that wasn't part of full review for freshness, please reset them to their previous value.</li></ul> | 
  | **Placement and linking** | If you're creating a new article or articles, include updates to the related TOC.yml file to propose where the article(s) should be placed. Also consider other places within the document set where it would be beneficial to cross-reference and link to your new article(s). | 
  | **PR build** | After you open your PR, and for each successive commit that you push to your branch, the publishing platform will run validation on the files in your pull request. A summary of the build results for each file will be inserted inline into your pull request, which includes any build suggestions/warnings/errors. PRs cannot be merged until all build errors and most warnings are resolved. |
  | **Publishing** | Following a successful merge, most repos publish to the live site at least once per (business) day, usually around 10am Pacific. |
  | **Additional resources** | <ul><li>[Learn.Microsoft.Com contributor guide](https://review.learn.microsoft.com/help/contribute/?branch=main)</li></ul>
  
## Additional guidance for private repos and internal contributors  

  | **Topic** | **Guidance** | 
  | ----------| ------------ | 
  | **PR size** | If your PR is more than ~5 lines of changes, or you'd like for the changes to go through editorial or larger review, open a contribution request at https://aka.ms/Contribution and include a link to the PR in response #8. Once it's processed, you'll be notified of the next steps.  |
  | **PR title prefix** | Select the **Edit** button to the right of the PR title if you need to revise it. The following prefixes are reserved for specific contribution types:<br/><br/><ul><li>**[Quality Check]** - maintenance work related to content quality (edit passes, art improvements, template alignment)</li><li>**[LinkFix]** - recurring/adhoc PRs to correct link URLs</li><li>**[Pipeline]** - new/updated contributor success pipeline content</li><li>**[WIP]** - a work-in-progress draft requiring several days/weeks</li></ul> |
  | **PR preview** | Following successful build of your PR, publishable files will also include **Preview URL** links to staged previews of your new/updated articles. Be sure to review these for verification of your intended contributions, or to send to other internal contributors for review. |
  | **PR sign-off (public repo)** | If an article you own is updated in a public repo PR, you are responsible for sign-off. You will be automatically notified via email. The PR will not be merged until you've had a chance to review and sign-off. |
  | **PR sign-off (private repo)** | After you've completed your proposed changes, addressed build warnings, and completed all review work, you can begin the sign-off process for review and merge:<br/><br/><ol><li>If your PR is in draft mode, remove "[WIP]" from the title and select **Ready for review** button at the bottom of the PR.</li><li>Enter "#sign-off" in a new comment. This comment indicates that **you're confident the work meets or exceeds Microsoft's standards for publication**, and will trigger the review process.</li><li>Your PR may be selected for initial review by the CACT. Following CACT review, you may receive questions or requests for additional changes. You should have initial feedback from CACT review within a few business days. If you have an urgent request or need to contact the team, please mention `@MicrosoftDocs/cloud-architecture-content-team-pr-reviewers` in your PR and someone will get back to you. After CACT review is complete, a `CACT #sign-off` will be added.</li><li>Final review/merge is done by the PR review team. The PR team may also respond with feedback, categorized as "Blocking" (requires action from you), or "Non-blocking" (to be addressed in a future PR).</li></ol> |
  | **Additional resources** | <ul><li>[Learn.Microsoft.Com internal contributor guide](https://review.learn.microsoft.com/help/contribute/?branch=main)</li><li>Authoring templates: [architecture-center-pr](https://review.learn.microsoft.com/help/contribute/architecture-center/templates/sample-solution-templates?branch=main), [well-architected-pr](https://review.learn.microsoft.com/help/contribute/global-waf-template?branch=main)</li><li>To contact the CACT use [e-mail](mailto:cact-pr-reviewers@microsoft.com?subject=Help%20with%20pull%20request), or @mention our GitHub team in your PR comments using: `@MicrosoftDocs/cloud-architecture-content-team-pr-reviewers`</li></ul>
</details>

### CODE REVIEW

**Comment by Court72** [comment]
@TomHickling

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

@MicrosoftDocs/public-repo-pr-review-team

**Comment by TomHickling** [comment]
This statement is also not true: "Persistent and Non-persistent terminology is used as reference to highlight the load balancing behavior for personal and pooled desktops"

Persistent and Non-persistent terminology is used to describe one of two things: 1. The persistence of the VM, i.e. it resets back to an original image, or it relates to user persistency, i.e. the user profile is stored separately from the VM itself and not persisted on the vm. What is your context

**Comment by dacarroll** [comment]
Hi @TomHickling - I may have mis-phrased my intended purpose here when submitting this PR. I agree with you that the statement I added isn't accurate.

However, my goal and context were that persistent and non-persistent is nearly always referenced in the context of the VM rather than just a user profile. Stating that a pooled VM that is OS persistent and will retain any changes made to it, but just because the person is unlikely to be load-balanced to that VM is confusing to most individuals without a VDI background. 

In particular I've had to field hours of dialog with folks working on AV products and other software stating the gold image needs my settings in them and then they point back to this documentation. I believe with the a minor tweak to verbiage we can more accurately highlight we are speaking non-persistence from a user profile perspective and not an OS disk perspective (currently).

**Comment by ckittel** [comment]
> I believe with the a minor tweak to verbiage we can more accurately highlight we are speaking non-persistence from a user profile perspective and not an OS disk perspective (currently).

Can you please update your PR to include that verbiage alteration?  Thanks.

**Comment by ShannonLeavitt** [comment]
@dacarroll - Could you take a look at the requests above?

**Comment by dacarroll** [comment]
@ckittel , @ShannonLeavitt - Apologies for the delay. Adjustments made.

**Review by ckittel — APPROVED** [review]

File: docs/example-scenario/azure-virtual-desktop/azure-virtual-desktop-content.md [hunk 1]
```diff
@@ -79,6 +79,9 @@ By using personal desktop solutions, sometimes called *persistent desktops*, use
 
 Pooled desktop solutions, also called *non-persistent desktops*, assign users to whichever session host is currently available, depending on the load-balancing algorithm. Because users don't always return to the same session host each time they connect, they have limited ability to customize the desktop environment and don't usually have administrator access.
 
+> [!NOTE]
+> Persistent and non-persistent terminology in this case is in reference to the persistence of the user profile. It does not imply that the operating system disk reverts to golden image or discards changes on reboot.
```
Comment (ckittel) [inline docs/example-scenario/azure-virtual-desktop/azure-virtual-desktop-content.md:83]:
```suggestion
> Persistent and non-persistent terminology in this case is in reference to the persistence of the user profile. It does not imply that the operating system disk reverts to a golden image or discards changes on reboot.
```

**Comment by ckittel** [comment]
@TomHickling - can you please re-review the updated text?

Withholding PnP sign-off until then.

**Review by TomHickling — APPROVED** [review]
Looks good to me

**Comment by ckittel** [comment]
PnP #sign-off

#remove-label:"do-not-merge"
#label:"ready-to-merge"

**Review by Court72 — APPROVED** [review]

**Pull request merged**